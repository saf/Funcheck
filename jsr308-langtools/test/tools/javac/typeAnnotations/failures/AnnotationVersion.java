/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary test that only java 7 allows type annotations
 * @author Mahmood Ali
 * @compile/fail/ref=AnnotationVersion.out -XDrawDiagnostics -source 1.6 AnnotationVersion.java
 */
class AnnotationVersion {
  public void method() @A { }
}

@interface A { }
