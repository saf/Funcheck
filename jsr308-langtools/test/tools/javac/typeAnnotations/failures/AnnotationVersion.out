AnnotationVersion.java:9:25: compiler.err.type.annotations.not.supported.in.source: 1.6
1 error
